projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -77,11 +94,12 @@ async function triageNewIssues( payload, octokit ) {
 
 	// Find Priority.
 	const priority = findPriority( body );
-	if ( null !== priority ) {
+	const hasPriorityLabel = await hasPriorityLabels( octokit, ownerLogin, name, number );
+	if ( null !== priority && ! hasPriorityLabel ) {
```

Comment:
Yoda is needed here, you think?

Reply:
It's not, just an old habit. :) Fixed in 8983aa3decb3f9bb6692761f5871c1eee3d554a3